ui/pages/Project/components/ProjectOverview.jsx

```diff
@@ -282,10 +285,26 @@ const Dataset = React.memo(({ project, samplesByType, user }) => {
     })
   }
 
+  Object.assign(params, { namespace: project.workspaceNamespace, name: project.workspaceName })
+
+  const addAnvilDataButton = (
+    <Modal
+      modalName="addAnvilDataset"
+      title="Add Anvil Dataset"
+      size="small"
+      trigger={<ButtonLink>Request Additional Data</ButtonLink>}
+    >
+      <LoadWorkspaceDataForm params={params} projectGuid={project.projectGuid} />
```

Comment:
constructing an object is generally a bad idea. For `LoadWorkspaceDataForm` you should change the param field names to `workspaceNamespace` and `workspaceName` and then you can pass `project` as `params` here

Reply:
I see what you mean. But if I change the param field names, I have to change the other use of `LoadWorkspaceDataForm` in the `LoadWorkspaceData.jsx`. In that case, I have to construct an object for the params.

Reply:
To change the param names for the main form you just need to change the actual names of the parameters in the route:
```
<Route path="/create_project_from_workspace/:namespace/:name" component={LoadWorkspaceData} />
```
changes to 
```
<Route path="/create_project_from_workspace/: workspaceNamespace/: workspaceName" component={LoadWorkspaceData} />
```